packages/terra-clinical-data-grid/CHANGELOG.md

```diff
@@ -12,6 +14,7 @@ Unreleased
 * Changed react-intl to be a peerDependency instead of a devDependency
 * Updated lower bound for react-intl to ^2.8.0 to be consistent with changes in terra-core and terra-framework
 
+
```

Comment:
Nit: remove extra line

Reply:
I removed the extra line. https://github.com/cerner/terra-clinical/pull/479/commits/c553bfcd94fdfdf84a6ca8753d311a56afa150fd